tests/integration/test_fim/test_files/test_whodata_policy_change/test_whodata_policy_change.py

```diff
@@ -0,0 +1,174 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: File Integrity Monitoring (FIM) system watches selected files in whodata mode, if the policies for those
+       files change during runtime, the monitoring mode changes to realtime. This tests check that when the
+       policies change, monitoring continues correctly in realtime and events are detected.
+
+components:
+    - fim
+
+suite: whodata_policy_change
+
+targets:
+    - agent
+
+daemons:
+    - wazuh-syscheckd
+
+os_platform:
+    - windows
+
+os_version:
+    - Windows 10
+    - Windows Server 2019
+    - Windows Server 2016
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/file-integrity/index.html
+
+pytest_args:
+    - fim_mode:
+        whodata: Implies real-time monitoring but adding the 'who-data' information.
+    - tier:
+        0: Only level 0 tests are performed, they check basic functionalities and are quick to perform.
+        1: Only level 1 tests are performed, they check functionalities of medium complexity.
+        2: Only level 2 tests are performed, they check advanced functionalities and are slow to perform.
+
+tags:
+    - fim_whodata_policy_change
+'''
+import os
+import time
+
+import pytest
+from wazuh_testing.tools import PREFIX, configuration
+from wazuh_testing.tools.monitoring import FileMonitor, generate_monitoring_callback
+from wazuh_testing.tools.local_actions import run_local_command_returning_output
+from wazuh_testing import global_parameters, LOG_FILE_PATH
+from wazuh_testing.modules import fim
+from wazuh_testing.modules.fim import event_monitor as evm
+from wazuh_testing.modules.fim import FIM_DEFAULT_LOCAL_INTERNAL_OPTIONS as local_internal_options
+from wazuh_testing.modules.fim.utils import regular_file_cud
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=1)]
+
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# Configuration and cases data
+test_cases_path = os.path.join(TEST_CASES_PATH, 'cases_whodata_policy_change.yaml')
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_whodata_policy_change.yaml')
+
+
+# variables
```

Comment:
```suggestion
# Variables
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3627/commits/0d431cc2ea9340d5f7d0aef41bc14acd57a26ff0